freqtrade/rpc/rpc_manager.py

```diff
@@ -0,0 +1,59 @@
+"""
+This module contains class to manage RPC communications (Telegram, Slack, ...)
+"""
+
+from freqtrade.logger import Logger
+from freqtrade.rpc.telegram import Telegram
+
+
+class RPCManager(object):
```

Comment:
I think it would make sense to move this class to `freqtrade/rpc/__init__.py`

Reply:
Personally, i like a class in aclass-specific file more than in the __init__.py file.

pep8 defines the __init__ file as a place to import your classes - but not define them (even though that is not specifically mentioned)